### DISCUSSION TITLE
Override clarity modal animation

### DISCUSSION THREAD

**Discussion by gayathrirajendran** [vmware/clarity#6206] [Q&A] [answered]
Is there a way to override the default animation with a custom one for clr-modal ?

**Comment by bbogdanov** [comment, marked as answer]
There is but is you have to look more in how to overwrite Angular animations cause this is what we use. 

May I ask what you situation is? Curious about what requires the animation to be changed.

**Reply by gayathrirajendran** [reply]
yes, angular animations is a given. So turn offs clarity's default modal animation and then add a custom animation is the suggestion right, will try that.

The situation is a custom design like a side panel that opens from the right, which has a11y spec as that of a modal except that it slides in from the right and slides back to where it came from. Considering the usage of modal for that. Hence the thought to employ a slide-in-out from the side animation in the place of modal's ease anime

**Reply by mathisscott** [reply]
FWIW, the Clarity Core library (@cds/core) will have a component like that. No ETA as of yet. But I wouldn't imagine it's too far off.
